README.md

```diff
@@ -9,36 +9,86 @@ To use the generic wallet refer to `signTransaction.py`, `getPublicKey.py` or Le
 # How to Install developer version
 ## Configuring Ledger Environment
 
-* Install Vagrant and Virtualbox on your machine
-* Run the following
+* Install Docker on your machines
+* Checkout the app builder repository
 
+Note the `./full` path on `docker build` differentiate from the previous version now under `legacy`
 ```
-git clone https://github.com/fix/ledger-vagrant
-cd ledger-vagrant
-vagrant up
+git clone https://github.com/LedgerHQ/ledger-app-builder.git
+cd ledger-app-builder
+sudo docker build -t ledger-app-builder:latest ./full
```

Comment:
I'm not sure this is what should be done.
Indeed user can rebuild the docker image if they wants. But they can also use is directly with:
`sudo docker run --user "$(id -u)":"$(id -g)" --rm -ti -v "$(realpath .):/app" ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-lite:latest`

Reply:
Correct that is an option. I didn't use the docker image. Reasons are below. I will ask the other EOS engineers what they think. 

We try to use software with a known source and the verified git-hub commits fit that. I didn't find any signatures for the docker image. 
```
docker trust inspect --pretty ledger-app-builder-lite:latest
```

In addition, I have only rebuilt the docker image from source. I've never tried the other approach. I would prefer to document the steps that I've practiced and used.

Reply:
Sure rebuilding the image works, it's a bit longer but less verifiable that's true.